.github/workflows/pkg-version-bump.yaml

```diff
@@ -18,29 +18,36 @@ jobs:
     name: Generate Token
     runs-on: ubuntu-latest
     outputs:
-      token: ${{ steps.generate_token.outputs.token }}
-      app-slug: ${{ steps.generate_token.outputs.app-slug }}
+      token: ${{ steps.generate-bot-token.outputs.token }}
+      app-slug: ${{ steps.generate-bot-token.outputs.app-slug }}
+      user-id: ${{ steps.get-bot-user-id.outputs.user-id }}
     steps:
       - name: Generate release bot app token
-        id: generate_token
+        id: generate-bot-token
         uses: actions/create-github-app-token@v1
         with:
           app-id: ${{ secrets.HIROSYSTEMS_PKG_RELEASE_BOT_APP_ID }}
           private-key: ${{ secrets.HIROSYSTEMS_PKG_RELEASE_BOT_APP_PEM }}
 
       - name: Get bot user ID
-        run: echo "user-id=$(gh api "/users/${{ steps.generate_token.outputs.app-slug }}[bot]" --jq .id)" >> "$GITHUB_OUTPUT"
+        id: get-bot-user-id
+        env:
+          GH_TOKEN: ${{ steps.generate-bot-token.outputs.token }}
+        run: echo "user-id=$(gh api "/users/${{ steps.generate-bot-token.outputs.app-slug }}[bot]" --jq .id)" >> "$GITHUB_OUTPUT"
 
   homebrew:
     name: Homebrew
-    needs: generate-token
+    needs:
+      - generate-token
```

Comment:
The `Winget` job has this as a single-string format, this is a list-string format, should we make these consistent?